service/src/main/java/bio/terra/tanagra/service/ServiceUtils.java

```diff
@@ -0,0 +1,11 @@
+package bio.terra.tanagra.service;
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+public class ServiceUtils {
```

Comment:
reason added - yes, its a wrapper class. but enables us to change it everywhere is needed instead of individual changes, and controls the length etc. 
i am not strongly opinionated. Can remove if insisted.

Reply:
reverted in https://github.com/DataBiosphere/tanagra/pull/1001